packages/preact/src/index.ts

```diff
@@ -152,7 +151,8 @@ hook(OptionsTypes.DIFF, (old, vnode) => {
 				// first Signal prop triggers creation/cleanup of the updater:
 				if (!updater) updater = getElementUpdater(vnode);
 				// track which props are Signals for precise updates:
-				updater._props.push(i);
+				updater._props.push({ _key: i, _signal: value });
+				props[i] = value.peek()
```

Comment:
Just checked and this is the only change needed to make the new test case pass.

Reply:
Yes, need to add a test with a subsequent update

Reply:
@marvinhagemeister peculiarly the test still fails and we never enter the `updater` function 😅